pos_payment_terminal/tests/__init__.py

```diff
@@ -0,0 +1 @@
+# from . import test_transactions
```

Comment:
to check

Reply:
also removed in https://github.com/akretion/pos/pull/20